WobblyStealthGame/Assets/Scripts/Utils/AnimatorHelper.cs

```diff
@@ -94,8 +94,8 @@ public enum LookMode {
             _animator.SetLookAtWeight(LookIKWeight, ((LeftArmIKWeight + RightArmIKWeight) / 2) * ChestMaxLookWeight, 1, 1, 0);
 
             Vector3 lookPos = Vector3.zero;
-            if (CurrentLookMode == LookMode.TARGET) lookPos = LookTarget.position;
-            if (CurrentLookMode == LookMode.POINT) lookPos = LookPoint;
+            if (CurrentLookMode == LookMode.TARGET && LookTarget) lookPos = LookTarget.position;
+            if (CurrentLookMode == LookMode.POINT && LookTarget) lookPos = LookPoint;
```

Comment:
```suggestion
            if (CurrentLookMode == LookMode.POINT) lookPos = LookPoint;
```

Alleen die andere gebruikt het